src/routes.ts

```diff
@@ -8,18 +8,36 @@ export const enum routeList {
   Entropy = 'Entropy',
   PasswordEntry = 'PasswordEntry',
   SeedPhrase = 'SeedPhrase',
-  Home = 'Home'
+  Identity = 'Identity',
+  Interactions = 'Interactions'
 }
 
 const navigationOptions = {
   header: null
 }
 
-export const Routes = StackNavigator({
-    Landing: { screen: Landing, navigationOptions },
-    Entropy: { screen: Entropy, navigationOptions},
-    PasswordEntry: { screen: PasswordEntry, navigationOptions },
-    SeedPhrase: { screen: SeedPhrase, navigationOptions },
-    Home: { screen: Home, navigationOptions }
+export const HomeRoutes = TabNavigator(
+  {
+  Identity: { screen: Identity },
+  Interactions: {screen: Interactions }
+  },
+  {
+    tabBarOptions: {
+      activeTintColor: 'purple',
+      labelStyle: {
+        fontSize: 20,
```

Comment:
Perhaps we can use some value from the JolocomTheme instead of the harcoded 20, might make it easier to adjust font sizes in the future.